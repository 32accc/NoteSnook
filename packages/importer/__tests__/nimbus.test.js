const nimbusnote = require("../providers/nimbusnote");
const utils = require("./utils");

test("convert simplenote backup zip file", () => {
  let file = utils.getFile("export-2021-08-27_10-11-56.zip");
  expect(nimbusnote.convert([file])).toMatchSnapshot("nimbus-export");
});
