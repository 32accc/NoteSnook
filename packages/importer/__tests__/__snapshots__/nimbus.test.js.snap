// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`convert simplenote backup zip file: nimbus-export 1`] = `
Array [
  Object {
    "content": Object {
      "data": "<div id=\\"note-editor\\" class=\\"note-container theme-light\\" style=\\"position: relative;\\">
<div class=\\"editor-body\\">
<div class=\\"export-mode nedit-root notranslate size-normal style-normal\\" id=\\"note-root\\"><div class=\\"editable-text paragraph indent-0\\" data-block-background=\\"transparent\\" style=\\"text-align:left\\">This is a new note that I am working on</div></div>
</div>
</div>",
      "type": "html",
    },
    "notebooks": Object {
      "notebook": "My Notes",
      "topic": "My Notes",
    },
    "title": "ringing the bell",
  },
  Object {
    "content": Object {
      "data": "<div id=\\"note-editor\\" class=\\"note-container theme-light\\" style=\\"position: relative;\\">
<div class=\\"editor-body\\">
<div class=\\"export-mode nedit-root notranslate size-normal style-normal\\" id=\\"note-root\\"><div class=\\"editable-text paragraph indent-0\\" data-block-background=\\"transparent\\" style=\\"text-align:left\\">This is a new note that I am working on</div></div>
</div>
</div>",
      "type": "html",
    },
    "notebooks": Object {
      "notebook": "My Notes",
      "topic": "My Notes",
    },
    "title": "Wow",
  },
  Object {
    "content": Object {
      "data": "<div id=\\"note-editor\\" class=\\"note-container theme-light\\" style=\\"position: relative;\\">
<div class=\\"editor-body\\">
<div class=\\"export-mode nedit-root notranslate size-normal style-normal\\" id=\\"note-root\\"><div class=\\"editable-text paragraph indent-0\\" data-block-background=\\"transparent\\" style=\\"text-align:left\\">This is a new note that I am working on</div></div>
</div>
</div>",
      "type": "html",
    },
    "notebooks": Object {
      "notebook": "My Notes",
      "topic": "My Notes",
    },
    "title": "This is a new day and I am writi",
  },
  Object {
    "content": Object {
      "data": "<div id=\\"note-editor\\" class=\\"note-container theme-light\\" style=\\"position: relative;\\">
<div class=\\"editor-body\\">
<div class=\\"export-mode nedit-root notranslate size-normal style-normal\\" id=\\"note-root\\"><div class=\\"editable-text paragraph indent-0\\" data-block-background=\\"transparent\\" style=\\"text-align:left\\">This is a new note that I am working on</div></div>
</div>
</div>",
      "type": "html",
    },
    "notebooks": Object {
      "notebook": "My Notes",
      "topic": "My Notes",
    },
    "title": "my note",
  },
  Object {
    "content": Object {
      "data": "<div id=\\"note-editor\\" class=\\"note-container theme-light\\" style=\\"position: relative;\\">
<div class=\\"editor-body\\">
<div class=\\"export-mode nedit-root notranslate size-normal style-normal\\" id=\\"note-root\\"><div class=\\"editable-text paragraph indent-0\\" data-block-background=\\"transparent\\" style=\\"text-align:left\\">This is a new note that I am working on</div></div>
</div>
</div>",
      "type": "html",
    },
    "notebooks": Object {
      "notebook": "Bingo",
      "topic": "Bingo",
    },
    "title": "Oh okay I am making a note now.",
  },
  Object {
    "content": Object {
      "data": "<div id=\\"note-editor\\" class=\\"note-container theme-light\\" style=\\"position: relative;\\">
<div class=\\"editor-body\\">
<div class=\\"export-mode nedit-root notranslate size-normal style-normal\\" id=\\"note-root\\"><div class=\\"editable-text paragraph indent-0\\" data-block-background=\\"transparent\\" style=\\"text-align:left\\">This is a new note that I am working on</div></div>
</div>
</div>",
      "type": "html",
    },
    "notebooks": Object {
      "notebook": "Bingo",
      "topic": "Rope",
    },
    "title": "New note",
  },
  Object {
    "content": Object {
      "data": "<div id=\\"note-editor\\" class=\\"note-container theme-light\\" style=\\"position: relative;\\">
<div class=\\"editor-body\\">
<div class=\\"export-mode nedit-root notranslate size-normal style-normal\\" id=\\"note-root\\"><div class=\\"editable-text paragraph indent-0\\" data-block-background=\\"transparent\\" style=\\"text-align:left\\">This is a new note that I am working on</div></div>
</div>
</div>",
      "type": "html",
    },
    "notebooks": Object {
      "notebook": "Bingo",
      "topic": "Rope",
    },
    "title": "New note",
  },
  Object {
    "content": Object {
      "data": "<div id=\\"note-editor\\" class=\\"note-container theme-light\\" style=\\"position: relative;\\">
<div class=\\"editor-body\\">
<div class=\\"export-mode nedit-root notranslate size-normal style-normal\\" id=\\"note-root\\"><div class=\\"editable-text paragraph indent-0\\" data-block-background=\\"transparent\\" style=\\"text-align:left\\">This is a new note that I am working on</div></div>
</div>
</div>",
      "type": "html",
    },
    "notebooks": Object {
      "notebook": "Bingo",
      "topic": "Rope",
    },
    "title": "home",
  },
  Object {
    "content": Object {
      "data": "<div id=\\"note-editor\\" class=\\"note-container theme-light\\" style=\\"position: relative;\\">
<div class=\\"editor-body\\">
<div class=\\"export-mode nedit-root notranslate size-normal style-normal\\" id=\\"note-root\\"><div class=\\"editable-text paragraph indent-0\\" data-block-background=\\"transparent\\" style=\\"text-align:left\\">This is a new note that I am working on</div></div>
</div>
</div>",
      "type": "html",
    },
    "notebooks": Object {
      "notebook": "Bingo",
      "topic": "Rope",
    },
    "title": "This is a new note that I am wor",
  },
]
`;
